<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>691</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Interpretable machine learning-aided metabolomic selection 
for enhanced tomato flavor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_1">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>80</y>
     <width>610</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interpretation Plot: Zone One</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_theme">
          <property name="text">
           <string>Theme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_theme"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_upload">
        <property name="text">
         <string>Upload data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_upload">
        <property name="text">
         <string>Click</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_rating">
        <property name="text">
         <string>Consumer rating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_rating"/>
      </item>
      <item>
       <widget class="QLabel" name="label_scope">
        <property name="text">
         <string>Interpretation scope</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_scope"/>
      </item>
      <item>
       <widget class="QLabel" name="label_featureOrSample">
        <property name="text">
         <string>Feature or sample</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_featureOrSample"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_output">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_output">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_image_2">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>385</y>
     <width>610</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interpretation Plot: Zone Two</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
